barista/src/main/java/com/markelliot/barista/Server.java

```diff
@@ -121,6 +123,12 @@ public Builder disableTls() {
             return this;
         }
 
+        public Builder setSslContext(SSLContext context) {
```

Comment:
everywhere else the builder uses. names rather than a setter
```suggestion
        public Builder sslContext(SSLContext context) {
```